qr-code-auth/README.md

```diff
@@ -34,7 +33,7 @@
         "data": {
           "user_id": "String",
           "device_info": "String",
-          "is_authorized": "Boolean",
+          "is_authorized": "String",
```

Comment:
What if the type was a union of 3 options that will be there and name was authorization_status?
Example: type Authorization_staus: "NOT_INIT" | "AUTHORIZED" | "REJECTED"
@shreya-mishra

Reply:
This way not any random string can be assigned to the variable. Enum can also be used here. But I had read somewhere that enums are not recommended in TS, so not 100% sure about those.

Reply:
Ref: https://www.youtube.com/watch?v=jjMbPt_H3RQ

Reply:
Firestore does not provide a direct method to restrict the values that can be stored in a field to a specific set of string values. we need to enforce the rule in the validator function to allow specific string values, and then store only those string values in the Firestore database. hence this field will be string and the validator function will allow only certain strings that can be defined in a list.